src/app/hooks/useMessage.ts

```diff
@@ -66,7 +70,7 @@ export const useMessage = (inputMessage: Message, mailSettings: any): [MessageEx
         transformWelcome,
         transformBlockquotes,
         transformStylesheet,
-        transformAttachements,
+        // transformAttachements,
```

Comment:
We should add a `TODO` to not miss it 😄

Reply:
This one is an oversight. I merged `transformEmbedded` and `transformAttachments` as it was mostly the same transformation in two steps.